### PR TITLE
Getem O&M Cost Calculator

### PR DESCRIPTION
**Pull request by mjprilliman**
-Add GETEM O&M cost calculator as ssc equation to call in UI calculations, PySAM. 
-Based on GETEM spreadsheet calculations
-Adds new cmod_geothermal_eqns.cpp

### CODE REVIEW

File: ssc/CMakeLists.txt [hunk 1]
```diff
@@ -16,8 +16,8 @@ set(SSC_SRC
 		cmod_battery.h
 		cmod_battery_eqns.cpp
 		cmod_battery_eqns.h
-        cmod_battery_stateful.cpp
-        cmod_battery_stateful.h
+        	cmod_battery_stateful.cpp
```
Comment (sjanzou) [inline ssc/CMakeLists.txt:19]: Not sure why these are shown tabbed over...

File: ssc/cmod_geothermal_costs.cpp [hunk 2]
```diff
@@ -778,10 +778,11 @@ class cm_geothermal_costs : public compute_module
         //OM Cost calculations
         /*
         double unit_plant = as_double("gross_output");
-        double cooling_water_flow_rate = as_double("cooling_water_flow_rate"); //todo provide input
-        double cooling_water_cost = as_double("cooling_water_cost"); //todo provide input
+        double cooling_water_flow_rate = as_double("cwflow"); //todo provide input
+        double cooling_water_cost = 300; //$/acre-ft //todo provide input
```
Comment (sjanzou) [inline ssc/cmod_geothermal_costs.cpp:782]: Are we going to get an input value from GTO or is the cooling water cost the same for all scenarios?

File: ssc/cmod_geothermal_costs.cpp [hunk 3]
```diff
@@ -796,7 +797,7 @@ class cm_geothermal_costs : public compute_module
         double plant_om = non_labor_om_cost + labor_om_cost;
 
         //Field OM
-        double percent_drilling_cost_om = as_double("percent_drilling_cost_om");
+        double percent_drilling_cost_om = 1.5 / 100.0; //% always
```
Comment (sjanzou) [inline ssc/cmod_geothermal_costs.cpp:800]: Is the drilling o and m finalized to a constant for all scenarios?

**Review by sjanzou — APPROVED** [review]
Please review the comments and applicability to all scenarios. Thanks!

File: ssc/cmod_geothermal_costs.cpp (hunk 3 above)
Comment (mjprilliman) [reply ssc/cmod_geothermal_costs.cpp:800]: Yes, for all relevant cases as confirmed by Erik.

File: ssc/cmod_geothermal_costs.cpp (hunk 2 above)
Comment (mjprilliman) [reply ssc/cmod_geothermal_costs.cpp:782]: Updated to be 1.7 * chemical_ppi for flash, overall costs goes to 0 for binary.

**Review by sjanzou — APPROVED** [review]
Looks good, thanks for updating

**Pull request merged**